src/img/icons/emoji-shooting-star.svg

```diff
@@ -0,0 +1,6 @@
+<svg viewBox="0 0 30 30">
```

Comment:
This SVG doesn't render properly for some reason:

![Screenshot from 2021-09-30 20-39-11](https://user-images.githubusercontent.com/1193222/135504162-94c36870-f19e-41f3-b290-b35fe68986ea.png)

Notice how it's smaller than the smite emoji. Please get a proper one from Karol (or the internet). If that doesn't work out, let me know, and I'll help

Reply:
Interesting, I think I know what it is, it should be simple to fix